lib/src/fields/search_field.dart

```diff
@@ -0,0 +1,236 @@
+import 'package:macos_ui/macos_ui.dart';
+import 'package:macos_ui/src/library.dart';
+import 'package:flutter/services.dart';
+
+/// A macOS-style search field.
+class MacosSearchField extends StatelessWidget {
```

Comment:
Will this widget contain a drop-down of search results in the future? At present, it functionally isn't different than a regular `MacosTextField`.

Reply:
Yes, the plan is to add a popup with search results, like when you search in the Finder app, and also a Toolbar-appropriate version, with the correct styling and show/hide behavior. Currently, it's a convenient wrapper around `MacosTextField` (in the same way as Material has TextField and TextFormField).

Reply:
Do you plan on doing that as part of this PR or separately?

Reply:
No, in a following PR soon, if that's ok, along with the rest of my notes above

Reply:
Okay, do you think you can have it ready by Wednesday?